src/tests/multihost/alltests/test_proxy_rfc2307bis.py

```diff
@@ -414,16 +381,9 @@ def test_simple_deny_groups_user_cs2_grp1(self, multihost, backupsssdconf):
                 client.login(login_timeout=10, sync_multiplier=1,
                              auto_prompt_reset=False)
             time.sleep(3)
-            execute_cmd(multihost, f'cat /var/log/secure | grep "Access denied for user {user}"')
+            check_logs(multihost, user)
         execute_cmd(multihost, "> /var/log/secure")
         execute_cmd(multihost, "sed -i 's/user_cs2_grp1/user_cs2_grp1_alias/' /etc/sssd/sssd.conf")
```

Comment:
Try not to use the command, instead of that use update method for sssd.conf file.